filament/src/materials/tonemapping.mat

```diff
@@ -6,25 +6,44 @@ material {
             name : colorBuffer,
             precision: high
         },
+        {
+            type : sampler2d,
+            name : bloomBuffer,
+            precision: medium
+        },
         {
             type : int,
             name : dithering
         },
         {
             type : int,
             name : fxaa
+        },
+        {
+            type : float2,
+            name : bloom
         }
     ],
+    variables : [
+        vertex
+    ],
     depthWrite : false,
     depthCulling : false,
     domain: postprocess
 }
 
+vertex {
+    void postProcessVertex(inout PostProcessVertexInputs postProcess) {
+        postProcess.vertex.xy = postProcess.normalizedUV;
+    }
+}
+
 fragment {
 
 #include "../../../shaders/src/tone_mapping.fs"
 #include "../../../shaders/src/conversion_functions.fs"
 #include "../../../shaders/src/dithering.fs"
+#include "../../../shaders/src/bloom.fs"
```

Comment:
Please add `bloom.fs` to the list in [`filament/CMakeLists.txt`](https://github.com/google/filament/blob/bedc6090515a187464f059d25a72cb34ea04820a/filament/CMakeLists.txt#L211). This establishes a dependency between `bloom.fs` and the tonemapping material.